CI/physmon/physmon.py

```diff
@@ -298,21 +306,27 @@
                 trackingGeometry,
                 field,
                 CKFPerformanceConfig(ptMin=400.0 * u.MeV, nMeasurementsMin=6),
+                TrackSelectorRanges(
+                    removeNeutral=True,
+                    absEta=(None, 2.5),
+                    loc0=(None, 4.0 * u.mm),
+                    pt=(500 * u.MeV, None),
+                ),
```

Comment:
Does this select tracks or particles?

Reply:
I just moved the track selection from the vertexing to CKF because it made more sense to me. similar to seeding where we also filter in the `addSeeding` command